config/plugins/interactive_environments/jupyter/config/jupyter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interactive_environment SYSTEM "../../interactive_environments.dtd">
+<interactive_environment name="Jupyter (Programming Environment)">
```

Comment:
How about `Jupyter (IPython)` since they use that occasionally to indicate the transitional nature of their project